### ISSUE TITLE
Julia Layer yields error

### ISSUE THREAD

**Issue by mpf** [syl20bnr/spacemacs#12852]
#### Description :octocat:
Entering `julia-mode` yields the error `File local-variables error: (void-function spacemacs//go-setup-backend)`

Requires `julia` layer

#### Reproduction guide :beetle:
- Start Emacs
- Open a `.jl` file

#### System Info :computer:
- OS: darwin
- Emacs: 26.2
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 58dcd6320)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(osx python helm
     (julia :variables julia-mode-enable-lsp t)
     lsp auto-completion
     (latex :variables latex-enable-auto-fill t latex-build-command "LaTeX")
     emacs-lisp git markdown treemacs)
```
- System configuration features: JPEG RSVG IMAGEMAGICK GLIB NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS LCMS2

**Comment by gdkrmr** [comment]
I can confirm this.

**Comment by mpf** [comment]
This worked for me. Thanks!

By the way, what is the correct way to set path for the Julia executable? Under `.spacemacs` layers, I have
```
 (julia :variables
            julia-mode-enable-lsp t
            lsp-julia-command "/usr/local/bin/julia"
            julia-mode-enable-ess nil)
```
but that doesn't seem to work.

**Comment by duianto** [comment]
The changes from PR: `fix #12852` #12854 
are now on the `develop` branch.
